eng/tools/RepoTasks/DownloadFile.cs

```diff
@@ -1,5 +1,5 @@
-// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
I don't think we want to change these.  We actually want to switch to the MIT license, so might be cleaner to just move there directly.

Reply:
@Pilchie Great. So I should attempt to make it MIT altogether right?

I see there are two forms of MIT license comments in source. Which form do you want?

Reply:
Actually, three forms:

1.

    ```csharp
    // Licensed to the .NET Foundation under one or more agreements.
    // The .NET Foundation licenses this file to you under the MIT license.
    // See the LICENSE file in the project root for more information.
    ```

2.

    ```csharp
    // Copyright (c) .NET Foundation and contributors. All rights reserved.
    // Licensed under the MIT license. See LICENSE file in the project root for full license information.
    ```

3.

    ```csharp
    // The .NET Foundation licenses this file to you under the MIT license.
    // See the LICENSE file in the project root for more information.

   ```

Reply:
@richlander - do you know the current preferred form of the license header for MIT?

Reply:
From Runtime:
https://github.com/dotnet/runtime/blob/a0d882641068c51146f1675806f6372d8ab0a3a5/.editorconfig#L167
>Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.

Reply:
Thanks. I'll reset the existing commits and match the header with dotnet/runtime, then force push.